firestore.rules

```diff
@@ -35,7 +35,7 @@ service cloud.firestore {
     }
     match /venues/{restOfPath=**} {
       allow read: if true;
-      allow write: if request.auth.uid != null;
+      allow write: if request.auth.uid in resource.data.owners;
```

Comment:
Thinking we should use a function with Firebase admin rather than letting the venue owner write any field, otherwise they could change the template etc.